proto/dclauth/pending_account.proto

```diff
@@ -11,7 +11,5 @@ message PendingAccount {
   // option (gogoproto.goproto_getters)  = false;
   // option (gogoproto.goproto_stringer) = false;
 
-  Account address = 1 [(gogoproto.embed) = true]; 
-  repeated string approvals = 2; 
-  
+  Account address = 1 [(gogoproto.embed) = true];   
```

Comment:
It was actually already there, but looks like a wrong name, should be ` Account account = 1`